# Fix hard-coded R and Perl paths
# Novermber 29th 2018 by B. Hajgato (Free University Brussels - VUB)
--- novocraft/IONTorrent.R.orig	2018-11-27 10:13:55.000000000 +0100
+++ novocraft/IONTorrent.R	2018-11-28 12:14:56.360220314 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/Rscript --no-save --no-restore  
+#!/usr/bin/env Rscript --no-save --no-restore  
 ## IONTorrent.R
 ## Colin Hercus
 ## SYNOPSIS: Plot Run Length error charts
--- novocraft/novo2paf.orig	2018-11-27 10:13:55.000000000 +0100
+++ novocraft/novo2paf	2018-11-28 12:14:27.959763232 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 use Getopt::Long;
 use Data::Dumper;
 
--- novocraft/novo2sam.pl.orig	2018-11-27 10:13:55.000000000 +0100
+++ novocraft/novo2sam.pl	2018-11-28 12:14:47.102071340 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 
 # Contact: lh3
 # Version: 0.2.0
--- novocraft/novorun.pl.orig	2018-11-27 10:13:55.000000000 +0100
+++ novocraft/novorun.pl	2018-11-28 12:14:40.673967889 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 
 use File::Basename;
 use Data::Dumper;
--- novocraft/qcalplot.R.orig	2018-11-27 10:13:55.000000000 +0100
+++ novocraft/qcalplot.R	2018-11-28 12:15:00.823292120 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/Rscript --no-save --no-restore  
+#!/usr/bin/env Rscript --no-save --no-restore  
 ## qcalplot.R
 ## Colin Hercus and Zayed Albertyn
 ## SYNOPSIS: Plot quality values 
--- novocraft/novope2bed.pl.orig	2018-11-27 10:13:55.000000000 +0100
+++ novocraft/novope2bed.pl	2018-12-03 09:55:07.644779954 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 
 use Getopt::Long;
 use Data::Dumper;
